diff --git "a/src\\gui\\image\\qjpeghandler.pri" "b/src\\gui\\image\\qjpeghandler.pri"
index de40c67..67ee739 100644
--- "a/src\\gui\\image\\qjpeghandler.pri"
+++ "b/src\\gui\\image\\qjpeghandler.pri"
@@ -3,7 +3,7 @@ HEADERS += $$PWD/qjpeghandler_p.h
 SOURCES += $$PWD/qjpeghandler.cpp
 contains(QT_CONFIG, system-jpeg) {
     msvc: \
-        LIBS += libjpeg.lib
+        LIBS += jpeg.lib
     else: \
         LIBS += -ljpeg
 } else {
