diff --git "a/src\\3rdparty\\png_dependency.pri" "b/src\\3rdparty\\png_dependency.pri"
index 78da861..8699615 100644
--- "a/src\\3rdparty\\png_dependency.pri"
+++ "b/src\\3rdparty\\png_dependency.pri"
@@ -1,6 +1,6 @@
 contains(QT_CONFIG, system-png) {
     unix|mingw: LIBS_PRIVATE += -lpng
-    else: LIBS += libpng.lib
+    else: LIBS += libpng16.lib
 } else: contains(QT_CONFIG, png) {
     include($$PWD/libpng.pri)
 }
